OsmAnd/res/values/strings.xml

```diff
@@ -715,7 +715,7 @@
     <string name="osm_editors_ranking">OSM Editors ranking</string>
     <string name="osm_live_subscription">OsmAnd Live subscription</string>
     <string name="osm_live_subscribe_btn">Subscribe</string>
-    <string name="osm_live_email_desc">Needed to provide you information about contributions.</string>
+    <string name="osm_live_email_desc">Needed to provide you info about contributions.</string>
```

Comment:
For a message like this, I think using the full word "information" is more appropriate.